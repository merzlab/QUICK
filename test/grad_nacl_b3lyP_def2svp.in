DFT B3LYP BASIS=DEF2-SVP cutoff=1.0e-9 denserms=1.0e-6 GRADIENT DIPOLE CHARGE=0

Na        -6.71296        1.52486        0.00000
Cl        -4.31558        1.52486        0.00000


