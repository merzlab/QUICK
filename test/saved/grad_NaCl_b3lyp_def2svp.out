  *********************************************************************************************************
  **                                                                                                     **
  **           888888888888                                                                              **
  **         8888888888888888                                                                            **
  **      888888888888888888                                                                             **
  **     888888888888888888                                                                              **
  **    888888888888888                                                                                  **
  **   88888888888888888888                               88888                       8888:              **
  **   8888888888888888888888Z                            88888                       8888:              **
  **   888888888888888888888888?                          88888                       8888:              **
  **   8888888888888      8888888                                        888888       8888:              **
  **    88888888888         888888     8888:     88888    88888        888888888I     8888:    888888    **
  **    8888888888           88888:    8888:     88888    88888      $888888888888    8888:   888888     **
  **    I8Z 88888             88888    8888:     88888    88888    .888888     8888   8888: 888888       **
  **    .8Z 88888             88888    8888:     88888    88888    $88888             8888:88888         **
  **     8I 88888      .=88. .88888    8888:     88888    88888    88888              8888888888         **
  **    :8  88888      888888$8888$    8888:     88888    88888    8888O              88888888888        **
  **   ,7   +88888.     8888888888.    8888:     88888    88888    888888             88888O888888       **
  **         $888888:.   .8888888      88888....888888    88888     888888     8888   8888:  888888      **
  **          I8888888888888888888     888888888888888    88888     O888888888888O    8888:   888888     **
  **            O888888888888888888     88888888888888    88888       88888888888$    8888:    888888    **
  **               8888888Z     888      .8888I  88888    88888         8888888       8888:     888888   **
  **                                                                                                     **
  **                                                                                                     **
  **                                         Copyright (c) 2021                                          **
  **                          Regents of the University of California San Diego                          **
  **                                    & Michigan State University                                      **
  **                                        All Rights Reserved.                                         **
  **                                                                                                     **
  **                   This software provided pursuant to a license agreement containing                 **
  **                   restrictions on its disclosure, duplication, and use. This software               **
  **                   contains confidential and proprietary information, and may not be                 **
  **                   extracted or distributed, in whole or in part, for any purpose                    **
  **                   whatsoever, without the express written permission of the authors.                **
  **                   This notice, and the associated author list, must be attached to                  **
  **                   all copies, or extracts, of this software. Any additional                         **
  **                   restrictions set forth in the license agreement also apply to this                **
  **                   software.                                                                         **
  *********************************************************************************************************
                                                                                                           
  Cite this work as:                                                                                       
  Manathunga, M.; Chi, J.; Cruzeiro, V.W.D.; Keipert, K.; Pekurovsky, D.; Mu, D.; Miao, Y.;He, X.; Ayers,K;
  Brothers, E.; GÃ¶tz, A.; Merz,K. M. QUICK-21.03 University of California San Diego, CA and Michigan State
  University, East Lansing, MI, 2021                                                                       
                                                                                                           
  If you have any comments or queries, please send us an email for technical support:                      
  quick.merzlab@gmail.com                                                                                  
                                                                                                           
| TASK STARTS ON: Tue Mar  9 15:19:34 2021
| INPUT FILE :    grad_NaCl_b3lyp_def2svp.in
| OUTPUT FILE:    grad_NaCl_b3lyp_def2svp.out
| DATE FILE  :    grad_NaCl_b3lyp_def2svp.dat
| BASIS SET PATH: /home/mmanathunga/fix109/basis

 @ Read Job And Atom

  KEYWORD=B3LYP  BASIS=DEF2-SVP CUTOFF=1.0E-9 DENSERMS=1.0E-6  ZMAKE GRADIENT DIPOLE CHARGE=0                                                                                                                     
 ============== JOB CARD =============
 METHOD = DENSITY FUNCTIONAL THEORY
 DENSITY FUNCTIONAL = B3LYP
 DIRECT SCF 
 SAD INITAL GUESS 
 USE DIIS SCF
 PRINT ENERGY EVERY CYCLE
 Z-MATRIX CONSTRUCTION
 DIPOLE
 STANDARD GRID = SG1
 GRADIENT CALCULATION
 MAX SCF CYCLES =    200
 MAX DIIS CYCLES =   10
 DELTA DENSITY START CYCLE = 1000
 COMPUTATIONAL CUTOFF: 
      TWO-e INTEGRAL   =  0.100E-08
      BASIS SET PRIME  =  0.100E-19
      MATRIX ELEMENTS  =  0.100E-09
      BASIS FUNCTION   =  0.100E-09
      GRADIENT CUTOFF  =  0.100E-08
 DENSITY MATRIX MAXIMUM RMS FOR CONVERGENCE  =  0.100E-05
 BASIS SET = DEF2-SVP,  TYPE = CARTESIAN
| BASIS FILE = /home/mmanathunga/fix109/basis/DEF2-SVP.BAS

 @ Finish reading job


 @ Begin SAD initial guess

 For Atom Kind =    1
 ELEMENT = NA
 BASIS FUNCTIONS =   16
 For Atom Kind =    2
 ELEMENT = CL
 BASIS FUNCTIONS =   19

 @ Finish SAD initial guess


 @ Begin Reading Molecular Information


 =========== Molecule Input ==========
 TOTAL MOLECULAR CHARGE  =    0    MULTIPLICITY                =    1
 TOTAL ATOM NUMBER       =    2    NUMBER OF ATOM TYPES        =    2
 NUMBER OF HYDROGEN ATOM =    0    NUMBER OF NON-HYDROGEN ATOM =    2
 NUMBER OF ELECTRONS     =   28

 -- INPUT GEOMETRY -- :
    NA         -6.7130       1.5249       0.0000
    CL         -4.3156       1.5249       0.0000

 -- DISTANCE MATRIX -- :
              1           2
      1     0.00000
      2     4.53039     0.00000

============== BASIS INFOS ==============
 BASIS FUNCTIONS =   35
 NSHELL =   15 NPRIM  =   71
 JSHELL =   15 JBASIS =   35


 @ End Reading Molecular Information 


 @ Begin XC Quadrature Formation

 OCTAGO: OCTree Algorithm for Grid Operations 
   PRUNING CUTOFF       = 0.100E-09
   INITIAL GRID POINTS  =        7520
|   FINAL GRID POINTS    =        5281
|   SIGNIFICANT NUMBER OF BASIS FUNCTIONS     =        2954
|   SIGNIFICANT NUMBER OF PRIMITIVE FUNCTIONS =        3402

 @ End XC Quadrature Formation


 @ Begin Energy Calculation

                                         SCF ENERGY
| ------------------------------------------------------------------------------------------------------------------------
| NCYC       ENERGY         DELTA_E      SCF_TIME  DII_CYC   DII_TIME   O_TIME  DIAG_TIME    MAX_ERR    RMS_CHG    MAX_CHG
| ------------------------------------------------------------------------------------------------------------------------
|  1   -622.286556555      ------         0.074     1        0.00      0.07      0.00    0.6206E+00  0.3024E-01  0.3425E+00
|  2   -622.345370877  0.588143E-01       0.076     2        0.00      0.08      0.00    0.1168E+00  0.4017E-01  0.5547E+00
|  3   -622.345159475  -.211402E-03       0.076     3        0.00      0.08      0.00    0.1120E+00  0.1968E-01  0.3272E+00
|  4   -622.389812725  0.446533E-01       0.076     4        0.00      0.08      0.00    0.5063E-02  0.1404E-02  0.1985E-01
|  5   -622.389884994  0.722692E-04       0.076     5        0.00      0.08      0.00    0.2977E-02  0.3603E-03  0.4823E-02
|  6   -622.389902433  0.174385E-04       0.076     6        0.00      0.08      0.00    0.5127E-03  0.1068E-03  0.1675E-02
|  7   -622.389903235  0.801651E-06       0.076     7        0.00      0.08      0.00    0.1353E-04  0.3214E-05  0.3011E-04
| -------------- 2E-INT CUTOFF CHANGE TO 0.1000E-08 ------------
|  8   -622.389903171  -.638412E-07       0.085     8        0.00      0.08      0.00    0.7853E-05  0.4246E-06  0.4666E-05
| ------------------------------------------------------------------------------------------------------------------------
| REACH CONVERGENCE AFTER   8 CYLCES
| MAX ERROR = 0.785313E-05   RMS CHANGE = 0.424622E-06   MAX CHANGE = 0.466585E-05
| -----------------------------------------------
 ALPHA ELECTRON DENSITY    =   13.9999387906
 BETA ELECTRON DENSITY     =   13.9999387906
 ELECTRONIC ENERGY    =  -663.666687830
 CORE_CORE REPULSION  =    41.276784659
 TOTAL ENERGY         =  -622.389903171

 @ End Energy calculation


 @ Begin Gradient Calculation

 ANALYTICAL GRADIENT: 
----------------------------------------
 COORDINATE    XYZ            GRADIENT
----------------------------------------
    1X    -6.7129605126    -0.0078419054
    1Y     1.5248601164    -0.0000000325
    1Z     0.0000000000    -0.0000000091
    2X    -4.3155803296     0.0080533539
    2Y     1.5248601164     0.0000000340
    2Z     0.0000000000     0.0000000096
----------------------------------------

 @ End Gradient Calculation


 @ Begin Zmake conversion

  
 Z-MATRIX:

   ATOM  ELEMENTAL  BOND LENGTH    BOND ANGLE    DIHEDRAL ANGLE
  NUMBER  SYMBOL    (ANGSTROMS)    (DEGREES)       (DEGREES)
     I                 I-NA         I-NA-NB        I-NA-NB-NC      NA   NB   NC

     1      NA       0.00000  0     0.00000  0      0.00000  0      0    0    0
     2      CL       2.39738  1     0.00000  0      0.00000  0      1    0    0

 @ End Zmake conversion


 @ Begin Charge and Dipole Calculation

    ATOMIC CHARGES
   ATOM          MULLIKEN            LOWDIN
   NA             0.5087             0.2400
   CL            -0.5087            -0.2400
   TOTAL         0.0000             0.0000

    DIPOLE (DEBYE)
      X         Y         Z        TOTAL
   -8.1093   -0.0000   -0.0000    8.1093

 @ End Charge and Dipole Calculation


 @ Output Timing Information

------------- TIMING ---------------
| INITIAL GUESS TIME  =     4.710360000( 82.44%)
| DFT GRID OPERATIONS =     0.016893000(  0.30%)
| TOTAL SCF TIME      =     0.619194000( 10.84%)
|       TOTAL OP TIME      =     0.616446000( 10.79%)
|             TOTAL 1e TIME      =     0.001991000(  0.03%)
|             TOTAL 2e TIME      =     0.407347000(  7.13%)
|             TOTAL EXC TIME     =     0.204979000(  3.59%)
|       TOTAL DII TIME      =     0.002683000(  0.05%)
|             TOTAL DIAG TIME    =     0.001038000(  0.02%)
| TOTAL GRADIENT TIME      =     0.361355000(  6.32%)
|       TOTAL 1e GRADIENT TIME      =     0.009428000( 0.18%)
|       TOTAL 2e GRADIENT TIME      =     0.291820000( 5.11%)
|       TOTAL EXC GRADIENT TIME     =     0.059355000(  1.04%)
| TOTAL TIME          =     5.713842000
------------------------------------
| Job cpu time:  0 days  0 hours  0 minutes  5.7 seconds.

 @ Finish Output Timing Information

| Normal Termination. Task Finished on: Tue Mar  9 15:19:40 2021
